### DISCUSSION TITLE
Happy new year

### DISCUSSION THREAD

**Discussion by CoffeeJackaboy** [Monika-After-Story/MonikaModDev#8815] [appreciation]
Happy new year DEVS! Thanks for making this year way better than it already was and thanks for making this mod .
(Also I don't want to sound greedy but can you add more features in the player appearance section like if the player has a beard (in the male situation) , if he has glasses , if he is skinny or fat , how he wears his hear and other stuff like that.)
Thanks again and Happy new year !

**Comment by CoffeeJackaboy** [comment]
It's hair not hear sorry.

**Comment by CloakedGhost** [comment]
Happy new year everyone! 

I don't think telling her your body type is going to be an option. Maybe I'm wrong, but some people could consider it offensive, you never know.

**Comment by multimokia** [comment]
Body type was actually originally brought up, but decided it was best not to include it. Discussion happened right on the PR for player appearance.
